### PR TITLE
docs: add Mac LaunchAgent instructions

### PR DESCRIPTION
**Pull request by cdaein**
From #999 

Instructions for Mac LaunchAgent so the Shiori server will start at login automatically.

Note:
I tested on my Mac but it would be nice if someone else can also do a test to make sure.
Thanks!

### CODE REVIEW

File: docs/Frequently-Asked-Question.md [hunk 1]
```diff
@@ -236,3 +236,34 @@ There are several methods to run `shiori` on start up, however the most recommen
     ```sh
     systemctl enable --now shiori
     ```
+
```
Comment (fmartingr) [inline docs/Frequently-Asked-Question.md:239]: I would add it as a separate section here:

```
### How to run `shiori` on start up (macOS)?
```

And rename the other one to specify Linux

**Review by fmartingr — CHANGES_REQUESTED** [review]
Thank you for the contribution! Really appreciated.

I think is best if we put the macOS setup under it's own section, I've left a comment about that. Just don't forget to update the table of contents and we're good to go :)

File: docs/Frequently-Asked-Question.md (hunk 1 above)
Comment (cdaein) [reply docs/Frequently-Asked-Question.md:240]: Made the requested changes. Please let me know if anything looks not ok. (I'm not very used to PR). Thanks.

**Review by fmartingr — APPROVED** [review]
This looks good! Thank you for your contribution. :)

**Pull request merged**